wavefront/server/apps/call_processing/call_processing/services/pipecat_service.py

```
@@ -519,6 +521,15 @@ async def run_conversation(
         task_container['task'] = task
 
         # Register event handlers
+        @llm.event_handler('on_function_calls_started')
+        async def on_function_calls_started(service, function_calls):
```

Ruff (0.15.2)

[warning] Unused function argument: `service`

(ARG001)